culturgen/util.py

```diff
@@ -127,15 +154,20 @@ def title_search(
             'query': query,
             'field': 'name',
             'fetch': 'name,url',
-            'len': 10,
+            'len': '10',
         },
     )
     try:
         data = r.json()
     except ValueError:
-        return {}
-
-    return {
-        item['name']: 'https://knowyourmeme.com' + item['url']
+        return
+
+    yield from (
+        TitleResult(
+            item['name'],
+            'https://knowyourmeme.com' + item['url'],
+            SequenceMatcher(None, query, item['name']).ratio(),
```

Comment:
More experiments with looking up memes like DYWMTCOAEYPTYCOMF raised an issue with using `SequenceMatcher` this way: It's case sensitive, i.e. `SequenceMatcher(None, 'a', 'A').ratio() == 0.0`.

This causes unintuitive false negatives when searching. Arguments to it should be `.casefold()`ed first.